src/main/resources/application.yml

```diff
@@ -127,6 +127,10 @@ prison-case-notes:
     - category: PRISON
       subcategory: RELEASE
 
+prison-adjudications:
+  lookback-days: 365
```

Comment:
Do we need adjudications from beyond then? I'd have thought we'd need adjudications from throughout the sentence? (Feel free to tell me if I've misunderstood something!)

Reply:
I think we agreed with Dan that a year was a good period for case notes, but you are probably right that a longer period is appropriate for adjudications. I think we need to make a  note to clarify this with Dan and others. 

But for now, the fact that the `lookbackDays` param is handled as config is probably sufficient?

Reply:
Good point - I'd assumed it would be the same as notes but perhaps not given the more serious nature of adjudications.  Will raise on Slack to see if anyone knows for sure.

Reply:
Looks back indefinitely now after feedback from Sarah on Slack